apps/src/studio/studio.js

```diff
@@ -153,6 +154,16 @@ function loadLevel() {
   // protagonistSpriteIndex was originally mispelled. accept either spelling.
   Studio.protagonistSpriteIndex = level.protagonistSpriteIndex || level.protaganistSpriteIndex;
 
+  switch (level.customGameType) {
+    case 'Big Game':
+      Studio.customGame = new BigGameInfo();
```

Comment:
Just for clarity, and it incidentally helps with RubyMine jump-to-definition and type annotating. We used it to define a minimal interface to conform with when passing the audio player down to blockly core to register and play sounds from there, and might be handy for stubbing out a player for tests later on. Probably a less meaningful usage of an interface than would be here since there will be multiple implementations soon-ish